boot.php

```diff
@@ -15,7 +15,7 @@
 use GameBoy\Keyboard;
 
 if (PHP_VERSION_ID >= 70000) {
-    set_exception_handler(function (Error $exception) {
+    set_exception_handler(function (Throwable $exception) {
```

Comment:
Why don't you use the global namespace here as you did on your previous PR?
Just to keep the same pattern.

Can you fix that?